packages/hs-react-ui/src/components/Label/Label.tsx

```diff
@@ -1,25 +1,83 @@
 import React from 'react';
 import styled from 'styled-components';
-
+import Icon from '@mdi/react';
+import { mdiCheckBold, mdiAsterisk } from '@mdi/js';
 import colors from '../../enums/colors';
 
 export const StyledLabel = styled.label`
-  ${({ color = colors.grayDark }: { color: colors | string }) => `
-    display: block;
+  ${({ color = colors.grayLight }: { color: colors | string }) => `
+    display: inline-flex;
     color: ${color};
-    text-transform: uppercase;
+    font-family: Arial, Helvetica, sans-serif;
     margin-bottom: .25em;
-    font-size: .75em;
+    font-size: .8em;
+  `}
+`;
+
+export const Child = styled.input`
+  ${({ color = colors.grayLight }: { color: colors | string }) => `
+  display: block;  
+  border: solid 2px ${color};
+  width: 15%;
   `}
 `;
```

Comment:
You should let them import whatever child they want (use the `children` prop) instead of making an input here.